sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -55,6 +57,7 @@ internal class AmqpConsumer : TransportConsumer
         ///
         private string PartitionId { get; }
```

Comment:
We are not using `partitionId` in Service Bus. Can we remove this?

Reply:
Removed